### PR TITLE
17028 Remove Dye & Durham references - Feature Flag

### PR DESCRIPTION
**Pull request by chenhongjing**
*Issue #:* /bcgov/entity#17028

*Description of changes:*
- Feature flagged to remove Dye & Durham references. ~A new feature flag `enable-specific-service-provider` should be added. Any advice for its name?~ A new feature flag `disable-specific-service-provider` should be added. Default value of this FF is `false`.
- Add replacement text. See below(*updated*).

image

Hyperlink in the screenshot: https://www.bcregistry.gov.bc.ca/filing

Further details can be seen here: https://docs.google.com/document/d/1RSuZ9PiOoNsdhaDKRYqHa6lL46djjDPJpUCl9OskQPU/edit?usp=drive_link

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Comment by chenhongjing** [comment]
I've just received email that the final version of replacement text is still under discussion. I'll update it later. 👀

File: queue_services/entity-emailer/src/entity_emailer/email_processors/ar_reminder_notification.py [hunk 1]
```diff
@@ -20,10 +20,14 @@
 from flask import current_app
 from jinja2 import Template
 from legal_api.models import Business, CorpType
+from legal_api.services.flags import Flags
 
 from entity_emailer.email_processors import get_recipient_from_auth, substitute_template_parts
 
 
+flags = Flags()  # pylint: disable=invalid-name
```
Comment (argush3) [inline queue_services/entity-emailer/src/entity_emailer/email_processors/ar_reminder_notification.py:28]: I think there are still some pieces missing.  In particular the pulling in of the LD sdk key and initialization of LD with this key.  

Take a look at how Vysakh does it for the AR reminder job.  It's structured a bit different so there will be differences in where we initialize LD.  https://github.com/bcgov/lear/pull/1817/files

image

File: queue_services/entity-emailer/flags.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "flagValues": {
+        "enable-specific-service-provider": true
+    }
+}
```
Comment (argush3) [inline queue_services/entity-emailer/flags.json:5]: As per our conversation, I think it makes sense to name the flag `disable-specific-service-provider` with a default value of `false`.  When we toggle the flag to stop serving the Dye & Durham text, this flag will toggle to `true`.

**Comment by chenhongjing** [comment]
Got the final version of replacement text and have updated it. 🚀

**Review by argush3 — APPROVED** [review]

**Pull request merged**